src/frontend/components/SøknadsSteg/Oppsummering/Oppsummering.tsx

```diff
@@ -288,6 +306,324 @@ const Oppsummering: React.FC = () => {
+                                        <SpråkTekst id={'ombarnet.oppholdutland.startdato.spm'} />
+                                    }
+                                    søknadsvar={
+                                        barn[barnDataKeySpørsmål.oppholdslandStartdato].svar
+                                    }
+                                />
+
+                                <OppsummeringFelt
+                                    tittel={
+                                        <SpråkTekst id={'ombarnet.oppholdutland.sluttdato.spm'} />
+                                    }
+                                    søknadsvar={
+                                        barn[barnDataKeySpørsmål.oppholdslandSluttdato].svar
+                                    }
+                                />
+                            </StyledOppsummeringsFeltGruppe>
+                        )}
+                        {barn[barnDataKeySpørsmål.boddMindreEnn12MndINorge].svar === ESvar.JA && (
+                            <StyledOppsummeringsFeltGruppe>
+                                <OppsummeringFelt
+                                    tittel={
+                                        <SpråkTekst
+                                            id={'ombarnet.sammenhengende-opphold'}
+                                            values={{ navn: barn.navn }}
+                                        />
+                                    }
+                                />
+                                <OppsummeringFelt
+                                    tittel={
+                                        <SpråkTekst
+                                            id={'ombarnet.sammenhengende-opphold.dato.spm'}
+                                        />
+                                    }
+                                    søknadsvar={
+                                        barn[barnDataKeySpørsmål.nårKomBarnTilNorgeDato].svar
+                                    }
+                                />
+                                <OppsummeringFelt
+                                    tittel={
+                                        <SpråkTekst
+                                            id={'ombarnet.planlagt-sammenhengende-opphold.spm'}
+                                        />
+                                    }
+                                    søknadsvar={
+                                        barn[barnDataKeySpørsmål.planleggerÅBoINorge12Mnd].svar
+                                    }
+                                />
+                            </StyledOppsummeringsFeltGruppe>
+                        )}
+                        {barn[barnDataKeySpørsmål.barnetrygdFraAnnetEøsland].svar && (
+                            <StyledOppsummeringsFeltGruppe>
+                                <OppsummeringFelt
+                                    tittel={
+                                        <SpråkTekst
```

Comment:
Litt usikker her, men skal dette spørsmålet rendres to ganger? Ser ut som det rendres lenger opp og?

Reply:
Ah, eller er det bare for å få overskriften kanskje?

Reply:
Det er for å få overskriften ja 😊